config/manifests/bases/dynatrace-monitoring.clusterserviceversion.yaml

```diff
@@ -0,0 +1,326 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Deep Insights
+    categories: Monitoring,Logging & Tracing,OpenShift Optional
+    containerImage: registry.connect.redhat.com/dynatrace/dynatrace-oneagent-operator
+    repository: https://github.com/Dynatrace/dynatrace-oneagent-operator
+    support: Dynatrace
+  name: dynatrace-monitoring.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
```

Comment:
I believe this gets generated and added on the generated CSVs, so no need to have it in this file.

Reply:
Tested and it didn't get added. Therefore I'll leave it

Reply:
The markers come from the the API markers from the DynaKube struct. So if they aren't getting added something is broken, and there is no point updating them on the code.